streams/writable-streams/aborting.js

```diff
@@ -204,34 +209,64 @@ promise_test(t => {
 }, 'Aborting a WritableStream puts it in an errored state, with a TypeError as the stored error');
 
 promise_test(t => {
-  const ws = new WritableStream();
+  let resolveWritePromise;
```

Comment:
The way in which tasks are processed by the browser is deterministically governed by web specifications. If your browser has a bug in processing tasks, then yeah, it can fail a lot of the tests. But we're not going to modify the tests for that purpose.